test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,39 @@
+require "test_helper"
+
+describe VotesController do
+  describe "create" do
+    before do
+      @user = perform_login
+      @work = works(:two)
```

Comment:
In response to your question on the comprehension questions: you don't _need_ to assign `@work = works(:two)` ... it's just an option, so that you don't have to do `works(:two)` each time (because `@work` is shorter?? I guess?? Also it vaguely reads more easily, so you don't have to think about syntax)

Reply:
But yeah, you're free to just do `works(:two)` the entire time